### ISSUE TITLE
MaxoutDense bias

### ISSUE THREAD

**Issue by hackiey** [fchollet/keras#5804]
I want to approximate a function f(x)=x^2 and I use the following code.

`model = Sequential()`
`model.add(MaxoutDense(output_dim=1,nb_feature=3,input_dim=1))`
`model.compile(loss='mae', optimizer=SGD(lr=0.001, momentum=0.4, nesterov=True))`
`model.fit(x_train, y, nb_epoch=20, batch_size=2)`

I referred to this article [Maxout Networks](http://www.simon-hohberg.de/2015/07/19/maxout.html), when I finished the training I thought there should be 3 lines, like the image in this article's example, but I just found 2 lines`f(x) = -6x` and `f(x) = 6x`, and all of them pass through the point (0,0). 

I guess there are 3 lines, I didn't find the 3rd line cause they don't have a bias term?

**Comment by bstriner** [comment]
`MaxoutDense` has a bias term. Check something `K.get_value(layer.W)` and `K.get_value(layer.b)` to see if the values make sense. Maybe try with a few more features and train a lot longer? If it doesn't look right please post code to reproduce.

Cheers

**Comment by hackiey** [comment]
thanks @bstriner, I will check them.
and why don't they support the maxoutdense in keras2.0?
